.changeset/fresh-crabs-greet.md

```diff
@@ -0,0 +1,6 @@
+---
+'@spotlightjs/spotlight': minor
```

Comment:
should this be `patch` instead since it's a bug fix?

Reply:
yes,

fixing this in new PR.